guidelines.md

```diff
@@ -62,6 +63,18 @@ In case of any error, the returned JSON object describes the error and has the f
 
 Some errors may also contain additional information relevant to the error on top of these two properties. That depends on the operation and is specifically described in the operation documentation.
 
+### Request limits
+
```

Comment:
Here I would write some general decription of why we have this and mentioned to contact us in case of issues. E.g. 

*In order to preserve stability of our API, we've put request limits in place. We continually strive to make the API more efficient, improve its usability, support batch operations and avoid the need of polling. However if you have any suggestion or use case that requires higher limits, please contact us at marketplace@mews.com.*

Reply:
added the text, though I'm not sure if we should mention the email address if we want to have integration partners use the chatbot. I do understand this could be more of a special case. @czech-man what do you think?